### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by DoraCretu594118**

### CODE REVIEW

**Comment by DoraCretu594118** [comment]
I don't like how responsiveness works, tried to improve this, if you have some ideas , share them pls

File: package.json [hunk 1]
```diff
@@ -9,7 +9,9 @@
   },
   "dependencies": {
     "axios": "^0.18.0",
+    "echarts": "^4.2.1",
     "register-service-worker": "^1.6.2",
+    "v-charts": "^1.19.0",
```
Comment (ecalcutin) [inline package.json]: Need to find a better alternative to this charts, or make it responsible & freezy-less

File: package.json [hunk 2]
```diff
@@ -9,7 +9,9 @@
   },
   "dependencies": {
     "axios": "^0.18.0",
+    "echarts": "^4.2.1",
```
Comment (ecalcutin) [inline package.json]: Need to find a better alternative to this charts, or make it responsible & freezy-less

File: src/layouts/dashboard/components/AppBar.vue [hunk 3]
```diff
@@ -1,44 +1,64 @@
 <template>
     <v-toolbar dark color="primary" class="customToolbar" app fixed clipped-left clipped-right>
+        <v-toolbar-title class="vp-title font-weight-black">Vita Pharm</v-toolbar-title>
```
Comment (ecalcutin) [inline src/layouts/dashboard/components/AppBar.vue]: App name looks too bold related to menu icon button

File: src/layouts/dashboard/components/Notification/NotificationBar.vue [hunk 4]
```diff
@@ -0,0 +1,92 @@
+<template>
+    <v-navigation-drawer
+        :value="notificationBar"
+        right
+        clipped
+        app
+        class="py-2 grey lighten-3 elevation-6"
+    >
+        <NotificationItem
+            :key="index"
```
Comment (ecalcutin) [inline src/layouts/dashboard/components/Notification/NotificationBar.vue]: remember to always provide KEY attribute in v-for cycle

File: src/layouts/dashboard/components/Notification/NotificationItem.vue [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+}
+
+.vp-title {
+    letter-spacing: -0.2px;
+}
+
+.vp-link:hover {
+    opacity: 0.8;
+}
+.vp-link:hover:before {
```
Comment (ecalcutin) [inline src/layouts/dashboard/components/Notification/NotificationItem.vue]: not sure whats happening here

Comment (DoraCretu594118) [reply src/layouts/dashboard/components/Notification/NotificationItem.vue]: overriding v-btn properties. we don't want to have any background on hover. If you find another solution for this ..

File: package.json (hunk 2 above)
Comment (DoraCretu594118) [reply package.json]: May not be responsive because I set a fixed width. Will try with max-width .

**Pull request merged**